prowler/providers/aws/services/iam/iam_inline_policy_allows_privilege_escalation/iam_inline_policy_allows_privilege_escalation.py

```diff
@@ -0,0 +1,48 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.iam.iam_client import iam_client
+from prowler.providers.aws.services.iam.lib.privilege_escalation import (
+    check_privilege_escalation,
+)
+
+
+class iam_inline_policy_allows_privilege_escalation(Check):
+    def execute(self) -> Check_Report_AWS:
+        findings = []
+
+        for policy in iam_client.policies:
+            if (
+                policy.attached or iam_client.provider.scan_unused_services
```

Comment:
Is this right? As per @sergargar comment here https://github.com/prowler-cloud/prowler/pull/4255/files#r1661478455